source/src/molecule-unfolding/cdk/utils/custom-resource-lambda/create-event-rule/template.json

```diff
@@ -0,0 +1,84 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "Cross region event bus for braket",
+    "Parameters": {
+        "TargetRegion": {
+            "Type": "String",
+            "Description": "Target Region",
+            "Default": "us-east-1"
+        }
+    },
+    "Resources": {
+        "EventRuleRegion1": {
+            "Type": "AWS::Events::Rule",
+            "Properties": {
+                "Description": "Routes to targetRegion event bus",
+                "EventBusName": "default",
+                "State": "ENABLED",
+                "EventPattern": {
+                    "source": [
+                        "aws.braket"
+                    ],
+                    "detail-type": [
+                        "Braket Task State Change"
+                    ]
+                },
+                "Targets": [
+                    {
+                        "Arn": {
+                            "Fn::Sub": "arn:aws:events:${TargetRegion}:${AWS::AccountId}:event-bus/default"
+                        },
+                        "Id": "CrossRegionDestinationBus",
+                        "RoleArn": {
+                            "Fn::GetAtt": [
+                                "EventBridgeIAMrole",
+                                "Arn"
+                            ]
+                        }
+                    }
+                ]
+            }
+        },
+        "EventBridgeIAMrole": {
+            "Type": "AWS::IAM::Role",
```

Comment:
why not moving creating this role to CDK? IAM role is independent to region.

Reply:
will change it to CDK